contrib/kind.sh

```diff
@@ -40,6 +40,7 @@ usage()
     echo "-i6 | --ipv6                 Enable IPv6. DEFAULT: IPv6 Disabled."
     echo "-wk | --num-workers          Number of worker nodes. DEFAULT: HA - 2 worker"
     echo "                             nodes and no HA - 0 worker nodes."
+    echo "-sm | --shared-mode          Enable shared gateway mode. DEFAULT: local."
     echo ""
```

Comment:
If we can do what I suggested from before, them L43 will be `echo "-gm | --gateway-mode"`

Reply:
I've done this in #1394, sorry I ended up opening a new PR, next time I'll just update existing ones.